todos/src/components/TimeStamp.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import { Time } from "styles";
+
+const timeStamp = (created) => {
+  const dateStamp = created.toDateString().split(' ').slice(1, -1).join(' ');
```

Comment:
Nice! I was looking at this and realising my date stamp is not working as I thought, will try this!